### DISCUSSION TITLE
iOS support?

### DISCUSSION THREAD

**Discussion by ljmatkins** [progrium/darwinkit#6] [Q&A]
This looks really cool!

Would this work for iOS as well?

**Comment by progrium** [comment]
as far as I know it should but i have not tried

**Comment by progrium** [comment]
The tricky bit is just figuring out the workflow for this. I'm confident a Go binary compiled for the right platform will run on iOS and it would use mostly the same frameworks. Until somebody does it and figures out the specifics I guess I'll say it's not supported.
